packages/firestore/test/util/test_platform.ts

```diff
@@ -87,7 +87,11 @@ export class FakeWindow {
  * `Document` fake that implements the `visibilitychange` API used by Firestore.
  */
 export class FakeDocument {
-  private _visibilityState: VisibilityState = 'unloaded';
+  /**
+   * unload is no more a valid member of VisibilityState, consider changing it to a valid value.
+   * https://developer.mozilla.org/en-US/docs/Web/API/Document/visibilityState
+   */
+  private _visibilityState: VisibilityState = 'unloaded' as VisibilityState;
```

Comment:
I think this was just a plain mistake.  Can you remove the comment you added and just change 'unloaded' to 'hidden'?  Thanks!

Reply:
Done.